lib/Backends/NNPI/Importer.cpp

```diff
@@ -1597,60 +1556,6 @@ class ChannelwiseQuantizedConvolutionNodeImporter : public INNPINodeImporter {
         glowChannelwiseQuantizedConv->getStrides()[1]};
     uint32_t dilation[SPATIAL_DIMS2] = {1, 1}; // No dilation, default values
 
-    // Add internal tensor for Symlowp input.
-    std::string symlowpInputName =
```

Comment:
Same

Reply:
In both cases the change is no longer needed (conversion to symlowp before PCQ).
That the support for gemmlowp inputs/outputs for PCQ conv that was enabled for Jack.